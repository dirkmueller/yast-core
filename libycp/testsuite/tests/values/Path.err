Parsed:
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
._._
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
.a0a9a._._._._.a0a9a
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
.modules.snd-card-ens1371
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(. + .example.path)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(.composed + .path)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((.three.times + .concatenated) + .path)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((.three.times + .concatenated) + .path)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(.three.times + (.concatenated + .path))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((.obscur + .example) + .)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((.obscur + .example) + .)
----------------------------------------------------------------------
