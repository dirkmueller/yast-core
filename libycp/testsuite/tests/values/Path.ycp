
# ---------------------------------------------------------
#
#  Filename:    Path.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------


// this tests includes tests for YCP_Void

# ------- YCP_Path
.
._._
.a0a9a._._._._.a0a9a

# paths may have '-' inside
# but not directly before or after '.'
# and '-' can not be the last character
# see PathErr[1234...].ycp for error cases

.modules.snd-card-ens1371

# ------- concatenating paths
( . + .example.path )
( .composed + .path )
( .three.times + .concatenated + .path )
( ( .three.times + .concatenated ) + .path )
( .three.times + ( .concatenated + .path ) )
( .obscur + .example + . )
(.obscur+.example+.)
