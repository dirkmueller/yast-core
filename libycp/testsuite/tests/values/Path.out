(.)
(._._)
(.a0a9a._._._._.a0a9a)
(.modules.snd-card-ens1371)
(.example.path)
(.composed.path)
(.three.times.concatenated.path)
(.three.times.concatenated.path)
(.three.times.concatenated.path)
(.obscur.example)
(.obscur.example)
